lib/urlwatch/main.py

```diff
@@ -109,5 +109,13 @@ def run_jobs(self):
         run_jobs(self)
 
     def close(self):
-        self.report.finish()
+        reported = self.report.finish()
+
+        if not reported:
+            logger.warning('No reporters enabled.')
```

Comment:
I like to do it like this mostly so that the "reported" local variable doesn't leak into the scope (this way, we can be sure that it isn't used below, since we don't give the expression result a name to begin with):

```suggestion
        if not self.report.finish():
            logger.warning('No reporters enabled.')
```